common/src/main/java/com/wynntils/core/consumers/features/FeatureCommands.java

```diff
@@ -44,4 +45,13 @@ public void discoverCommands(Feature feature) {
             }
         }
     }
+
+    public LiteralCommandNode<CommandSourceStack> getCommandNode() {
+        // Build the command node if it hasn't been built yet
+        if (commandNode == null) {
```

Comment:
I think this feels a bit shaky. What if someone calls this too early? Then the builder will build, and the next `commandNodeBuilder.then` will fail (I assume). 

While slightly more complicated, I suggest adding some kind of finalizing method here, to build the commandNode, and call it in the end of FeatureManager.init().

Reply:
Okay, sure.